learn/index.html

```diff
@@ -0,0 +1,16 @@
+---
+title: Learning Resources
+layout: default
+---
+<div class="pdb-50 pdt-50">
+  <p class="title">{{page.title}}</p>
+  {% for resource in site.data.learn %}
+    <div class="card">
+      <header class="card-header">
+        <p class="card-header-title">
+          <a href="{{resource.link}}" target="_blank">{{ resource.name }}</a>
```

Comment:
Let's add a small description for the sites on the side as well

Reply:
@zoras  dai, Done. Please take a look :)